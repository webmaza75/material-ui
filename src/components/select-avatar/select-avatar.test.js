import * as React from 'react';
import * as ShallowRenderer from 'react-test-renderer/shallow';

import SelectAvatar from './select-avatar';

describe(`SelectAvatar`, () => {
  it(`SelectAvatar renders correctly with not select mode`, () => {
    const renderer = ShallowRenderer.createRenderer();
    const isSelectMode = false;
    const selected = false;
    const type = 'Payload Monitoring Report';
    const result = renderer
      .render(<SelectAvatar
        isSelectMode={isSelectMode}
        selected={selected}
        type={type}
      />);

    expect(result).toMatchSnapshot();
  });

  it(`SelectAvatar renders correctly with non-selected item`, () => {
    const renderer = ShallowRenderer.createRenderer();
    const isSelectMode = true;
    const selected = true;
    const type = 'Payload Monitoring Report';
    const result = renderer
    .render(<SelectAvatar
      isSelectMode={isSelectMode}
      selected={selected}
      type={type}
    />);

    expect(result).toMatchSnapshot();
  });
});
