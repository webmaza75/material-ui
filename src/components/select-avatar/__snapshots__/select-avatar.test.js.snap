// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectAvatar SelectAvatar renders correctly with non-selected item 1`] = `
<SelectTaskAvatar
  isSelectedItem={true}
/>
`;

exports[`SelectAvatar SelectAvatar renders correctly with not select mode 1`] = `
<TaskAvatar
  type="Payload Monitoring Report"
/>
`;
